core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -322,21 +322,24 @@ object AppType {
     override def toString: String = "WDS"
   }
 
+  case object HailBatch extends AppType {
+    override def toString: String = "HAIL_BATCH"
+  }
+
   case object Custom extends AppType {
     override def toString: String = "CUSTOM"
   }
 
   def values: Set[AppType] = sealerate.values[AppType]
   def stringToObject: Map[String, AppType] = values.map(v => v.toString -> v).toMap
 
-  /** Host formatting for an App. Currently, for Azure there is only 1 configuration,
-   * and FormattedBy.Cromwell is expected. */
+  /** Disk formatting for an App. Currently, for Azure there is only 1 configuration,
+   * and FormattedBy.Cromwell is expected for WDS and HailBatch. */
```

Comment:
So this means that a disk used for a Cromwell app can be reused for a WDS or Hail app as well, I am just curious as why this is a desired behavior?
I guess what I am asking is why we cannot share between Galaxy and cromwell, but it is okay to share between cromwell and WDS/Hail

Reply:
That is a good point. I honestly didn't think too hard about this, just kind of copied the approach WDS was using.

It's a little moot because we're not doing any disk management for Azure apps anyway -- e.g. we don't support plugging in an existing disk to an Azure app. So maybe it makes sense not to impose a lot of restrictions just yet. It's a little unfortunate the code is structured in a way to make us define the disk behavior.

Reply:
I guess the only app types which support disk management (on GCP) is `Galaxy` and `Custom`. So maybe the 3 disk categories should be `Galaxy, Custom, Everything Else`

Reply:
Updated comment a bit to make this clear